### DISCUSSION TITLE
Which other RDBMs are planned?

### DISCUSSION THREAD

**Discussion by mamcx** [SeaQL/sea-query#181] [General]
I'm evaluating if get out my custom ORM and use this. I have the uncommon requirement that I actually use a lot of RDBMS (I integrate with dozens of ERPs), so wonder if exists some planning to which other RDBMs to extend this library. In order, for me:

- PG
- Sqlite
- MySql
- SQL Server
- Firebird
- ODBC and/or OLEDB (here I cover the other 5)

I know exist drivers in Rust for SQL Server/firebird. 

P.D: I could help with SQL Server that is probably the biggest priority for now, but needs to know if exist interest and if exist anything that could be a block, like CI support or similar...

**Comment by tyt2y3** [comment]
We want to support all open source databases, but have not decided yet for proprietary ones (e.g. MS SQL)
